staking_deposit/settings.py

```diff
@@ -11,14 +12,15 @@ class BaseChainSetting(NamedTuple):
 
 
 LUKSO = 'lukso'
+LUKSO_MAINNET = 'lukso-mainnet'
```

Comment:
I think that we should stick to `lukso` name for LUKSO Mainnet (fyi @frozeman)

Reply:
This is only an alias for lukso so that both lukso and lukso-mainnet will work correctly. I also switched the lookup to an exact match only

Reply:
Ok, nice.